build.sbt

```diff
@@ -14,18 +14,24 @@ developers := List(
 
 sbtPlugin := true
 
+crossSbtVersions := Vector("0.13.15", "1.0.0-M6")
+
 scalacOptions ++= List(
   "-unchecked",
   "-deprecation",
   "-Xlint",
   "-language:_",
-  "-target:jvm-1.6",
   "-encoding", "UTF-8"
-)
+) ++ (if (scalaVersion.value startsWith "2.10.") List("-target:jvm-1.6") else List.empty)
 
 resolvers ++= Seq(sonatypeSnapshots, sonatypeReleases)
 
-libraryDependencies += "org.scalariform" %% "scalariform" % "0.1.8"
+libraryDependencies += {
+  if (scalaVersion.value startsWith "2.10.")
+    "org.scalariform" %% "scalariform" % "0.1.8"
```

Comment:
Yeah, seems I failed to properly parse your last commend, sorry. Update on its way.

I was trying to look for [0.2.0 for 2.10](http://search.maven.org/#search%7Cga%7C1%7Ca%3Ascalariform_2.10) on Maven Central but I suppose its indexing hasn't caught up yet or something.